andela/boondocks-bn-backend
# #169257387 Integrate TravisCI with readme badge

#### What does this PR do?
This pull request sets up Integrates TravisCI with the readme badge for continuous integration

#### Description of Task to be completed
- [x] Connect the project to [Travis CI website](https://travis-ci.org/)
- [x] Configure the project to tell which version of node and language to be used in `.travis.yml` file
- [x] Place Travis CI badge on README file

#### How should this be manually tested?
- At the point of raising a Pull Request, the Travis will verify if the commit can be integrated into the existing repo by showing a green sign if it can, or red if it can not
- On the README a badge with a black-green colour will show if the commit is verified to be integrated

#### Any background context you want to provide?
This is required to automatically run tests and other preliminary build steps required to ship the next iteration of the product

#### What are the relevant pivotal tracker stories?
[#169257387](https://www.pivotaltracker.com/story/show/169257387)

#### Screenshots (if appropriate)
N/A
